saleor/order/models.py

```diff
@@ -82,7 +82,18 @@ def ready_to_confirm(self):
         return self.filter(status=OrderStatus.UNCONFIRMED)
 
 
+def get_order_number():
+    with connection.cursor() as cursor:
+        cursor.execute("SELECT nextval('order_order_number_seq')")
+        result = cursor.fetchone()
+        return result[0]
```

Comment:
This may cause races, surely there is a better way than doing this. We should leave it to postgres to do the auto-increment rather than making a SQL query to get the next value.

Maybe `BigAutoField` for `bigserial`? Hopefully it supports non-primary key fields https://github.com/django/django/blob/fac3dd7f390d372736e05974cc5c3ef1a3768fbb/django/db/backends/postgresql/base.py#L75-L76